README.rst

```diff
@@ -180,7 +185,7 @@ And browse http://127.0.0.1:5001:
    :width: 600px
 
 
-For more information, see `STAC REST interface usage <https://eodag.readthedocs.io/en/latest/use.html#stac-rest-interface>`_.
+For more information, see `STAC REST API usage <https://eodag.readthedocs.io/en/stable/stac_rest.html>`_.
 
 Command line interface
 ----------------------
```

Comment:
To be replaced with :

> Start playing with the CLI:
> - To search for some products:
```bash
eodag search --productType S2_MSI_L1C --box 1 43 2 44 --start 2021-03-01 --end 2021-03-31
```
> The request above searches for `S2_MSI_L1C` product types in a given bounding box, in March 2021. It saves the results in a GeoJSON file (`search_results.geojson` by default).

> Results are paginated, you may want to get all pages at once with `--all`, or search products having maximum 20% of coud cover with `--cloudCover 20`. For more information on available options:

```bash
eodag search --help
```
> - To download the result of the previous call to search:

```bash
eodag download --search-results search_results.geojson
```

> - To list all available product types and supported providers:
```
eodag list
```
>- To list available product types on a specified supported provider:
```
eodag list -p sobloo
```
>- To see all the available options and commands:
```
eodag --help
```
> - To print log messages, add `-v` to eodag master command. e.g. `eodag -v list`. The more `v` given (up to 3), the more verbose the tool is. For a full verbose output, do for example: `eodag -vvv list`

Reply:
Done in https://github.com/CS-SI/eodag/pull/233/commits/d507c08d46d13d199d80a757cc5e46544b718cfb